<launch>
    <arg name="test_case" default="minimal" />
    <arg name="debug" default="false" />

    <node pkg="capabilities" name="capability_server" type="capability_server" output="screen">
        <env name="ROS_PACKAGE_PATH" value="$(find capabilities)/test/discovery_workspaces/$(arg test_case):$(env ROS_PACKAGE_PATH)" />
        <param name="debug" value="true" if="$(arg debug)"/>
    </node>
</launch>
